test/e2e/toggle-task/03-update.yaml

```diff
@@ -1,5 +1,5 @@
 apiVersion: kudo.dev/v1beta1
 kind: TestStep
 commands:
-  - command: kubectl kudo update --instance feature-operator-instance -p enable_cm=false
-    namespaced: true
+  - command: kubectl kudo update --instance feature-operator-instance -p enable_cm=true
```

Comment:
This diff is confusing: I had to insert the `00-assert.yaml` at the beginning and so all other step files had their numbers increased and now it looks like there was a change here, but there wasn't.